bfxapi/rest/BfxRest.py

```diff
@@ -326,6 +329,40 @@ def __init__(self, API_KEY, API_SECRET, host='https://api.bitfinex.com/v2', loop
         credits = await self.post(endpoint, params=params)
         return [FundingCredit.from_raw_credit(c) for c in credits]
 
+    async def get_currency_moevements(self, currency, start, end, limit=25):
```

Comment:
I don't think this `get_all_movements` belongs to the library - let's keep it in the application